### ISSUE TITLE
Wallet not depositing to DOA

### ISSUE THREAD

**Issue by brain-dump** [nervosnetwork/neuron#2377]
I have been depositing to the DOA from Neuron every few days for the past 6 months.  As of a few days ago, perhaps the day of the fork, I can no longer deposit to the DAO.  All deposits sit on "pending".  I have cleared cache multiple times yet am still not able to deposit.  I also updated my wallet the the latest GitHub version as a possible resolution.  No joy.

Neuron Version: 0.101.3
CKB Client Version: v0.101.4

<img width="1343" alt="Screen Shot 2022-05-11 at 6 48 18 PM" src="https://user-images.githubusercontent.com/6054416/167960073-590b1a46-8510-41f3-876d-4798be76f428.png">

[neuron_debug_1652309190322.zip](https://github.com/nervosnetwork/neuron/files/8673811/neuron_debug_1652309190322.zip)

**Comment by Keith-CY** [comment]
Hi @brain-dump 

From the logs I found the following message
```
[2022-05-11 18:41:11.559] [warn] API Controller:	channel handling error: Error: Insufficient balance. Error: Insufficient balance.
    at Function.TransactionGenerator.generateDepositAllTx (/Applications/Neuron 2.app/Contents/Resources/app.asar/dist/services/tx/transaction-generator.js:811:15)
```

This is expected because we'll try to generate a deposit all to dao transaction on entering this tab.

To debug your case, I need more details about the pending transaction, could you please open the transaction detail window with the button in history
![image](https://user-images.githubusercontent.com/7271329/167986493-b29ef1d3-e5aa-4bfa-928b-aff1708d45cd.png)

And copy all inputs/outputs addresses
![image](https://user-images.githubusercontent.com/7271329/167986545-5b730e1e-37a4-421d-94a1-93dca2c3250f.png)

With those we can construct the transaction locally

These info maybe sensitive so you can send them to neuron@magickbase.com

**Comment by brain-dump** [comment]
Thanks for the quick response.  Sending email now.

**Comment by Keith-CY** [comment]
> Thanks for the quick response. Sending email now.

Got the transaction detail and we will try our best to fix the error.

**Comment by yanguoyu** [comment]
I didn't find your deposit transaction at the mainnet. I think it was not broadcast to the mainnet. Could you try to clear cache by `preferences-> General -> Clear Cache`? And then tell me the result.

**Comment by Keith-CY** [comment]
> I didn't find your deposit transaction at the mainnet. I think it was not broadcast to the mainnet. Could you try to clear cache by `preferences-> General -> Clear Cache`? And then tell me the result.

As the first message says, Neuron's cache has been cleared multiple times.

I can reproduce this pending deposit with release Neuron@v0.101.3 twice, we could checkout that version and debug in dev mode.

Although you said you have tried the latest Neuron(v0.103.0) and it didn't work, I still suggest you have a try again. I switched between v0.101.3 and v0.103.0 and found the following,

1. I deposit with v0.103.0 and it works
2. I deposit with v0.101.3 at **2022-05-14 15:25:20.003** and it is pending for a long time, then I relaunch with v0.103.0, the pending one is confirmed and can be found on chain, but the time of submission is **2022-05-14 15:30:59**, I switched back to v0.101.3 again, the dao record is confirmed too.
3. I deposit with v0.101.3 again at **2022-05-14 15:32:37**, and the tx is pending for **10mins**. Now I switch to v0.103.0, the pending one is confirmed at **2022-05-14 15:42:54** (sorry for forgetting to record the video, it turns too quickly).

From the actions I did, I think the tx generated by neuron is correct(the tx you emailed me is correct too as I constructed it manually) and submitted to ckb node inside neuron, but was not broadcasted successfully.

This may be related to hardfork that other nodes activated hardfork don't accept transactions from nodes not activated hardfork, I will check the possibility with ckb core team. @brain-dump
